File: testprojects/master_scala_main/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/Script_4.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_4 {
  def apply(spark: SparkSession, in0: DataFrame): Unit = {
    var out=in0.filter(col("c_short")  > -1)
    print(out.show())
  }

}

File: testprojects/python_main/pipelines/PYTHON_BASIC/code/pythonbasic/test/mainone/graph/Script_4.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pythonbasic.test.mainone.config.ConfigStore import *
from pythonbasic.test.mainone.udfs.UDFs import *

def Script_4(spark: SparkSession, in0: DataFrame) -> DataFrame:
    print("hello")
    out0 = in0

    return out0
